cfgov/tccp/jinja2/tccp/landing_page.html

```diff
@@ -51,7 +51,7 @@ <h2>
                 </li>
                 <li class="m-list_item m-list_item__has-icon">
                     <a
-                        href="/data-research/credit-card-data/terms-credit-card-plans-survey/"
+                        href="/consumer-tools/credit-cards/explore-cards/about/"
```

Comment:
There's a better Django way to do this that avoids hardcoding the URL like this:

```suggestion
                        href="{{ url('tccp:about') }}"
```